ui/app/projects/projects.module.js

```diff
@@ -42,6 +42,19 @@
         }
       });
 
+      var getSortFromLocalStorage = function () {
+        const DEFAULT_FILED = 'name';
+        const DEFAULT_ASC = 'asc';
+
+        var sortingConfigJson = window.localStorage.getItem('projectListSortingConfig');
```

Comment:
`projectListSortingConfig` constant should be defined only in a single place, but it could be again refactored separately as a part of NCL-5347

Reply:
+1

Reply:
Will fix it in NCL-5347